beacon-chain/db/slasherkv/slasher.go

```diff
@@ -259,14 +259,25 @@ func (s *Store) AttestationRecordForValidator(
 // then only the first one is (arbitrarily) saved in the `attestationDataRootsBucket` bucket.
 func (s *Store) SaveAttestationRecordsForValidators(
 	ctx context.Context,
-	attestations []*slashertypes.IndexedAttestationWrapper,
+	attWrappers []*slashertypes.IndexedAttestationWrapper,
 ) error {
 	_, span := trace.StartSpan(ctx, "BeaconDB.SaveAttestationRecordsForValidators")
 	defer span.End()
-	encodedTargetEpoch := make([][]byte, len(attestations))
-	encodedRecords := make([][]byte, len(attestations))
 
-	for i, attestation := range attestations {
+	const batchSize = 10_000
```

Comment:
This would be hard to find if we're searching for how this is controlled. Should we add some comments or give it a more meaningful name ?